deploy/environments/applications_viz.yaml

```diff
@@ -5,21 +5,15 @@ spack:
     tcl:
       whitelist:
         - brayns
-        - brayns-brainatlas
-        - brayns-topologyviewer
         - brion+python
         - meshball
         - ultraliser
   specs:
     - brayns%gcc +brion ^brion@3.3.1 ^ospray%intel
     - brayns@1.0.1%gcc +brion ^brion@3.1.0 ^ospray%intel
-    - brayns-brainatlas@0.1.0%gcc ^brayns@0.8.0 ^ospray@1.7.3%intel
-    - brayns-membranelessorganelles@0.1.0%gcc ^brayns@0.8.0 ^ospray@1.7.3%intel
-    - brayns@0.8.0%gcc +opendeck +brion ^brion@3.1.0 ^ospray@1.7.3%intel
     - brion@3.1.0 +python
     - brion@3.2.0 +python
```

Comment:
Can we clean some of these, too?

Reply:
I'd do it in a separate PR. As for brion@3.1.0 we will get rid of it if we can build brayns@1.0.1 with it.
I think it might require confirmation from other parties as well.

Reply:
`brion@3.1.0` can be removed as an explicit spec, but still be a dependency of `brayns@1.0.1`… unless you require the module to build it manually

Reply:
Fine to do it later, though